src/styles/main.scss

```diff
@@ -0,0 +1,30 @@
+.container {
+  --column-count: 1;
+
+  min-width: 1024px;
+  max-width: 1200px;
```

Comment:
The `min-width` and `max-width` properties should be avoided for `.container`. Instead, use `width` with a percentage value and `max-width` to limit the container width. This approach is more responsive and flexible.